pyproject.toml

```
@@ -23,12 +23,12 @@ packages = [
 yato = 'yato.cli:cli'
 
 [tool.poetry.dependencies]
-python = ">=3.8.1,<3.13"
+python = ">=3.9,<3.13"
 boto3 = "^1.34.51"
-sqlglot = "^22.0.1"
-duckdb = "^0.9.2"
+sqlglot = "^23.0.2"
+duckdb = "^0.10.0"
```

Hi @MohamedBsh, a new version has arrived! Very useful for using Yato with MotherDuck and for developers on Windows 🙂 
```suggestion
duckdb = "^1.0.0"
```

---

Thank you @jbondu9 . Your suggestion is relevant and has been implemented. DuckDB has been updated to version 1.0.0 in the pyproject.toml file 🤗